mne_bids/read.py

```diff
@@ -76,6 +76,42 @@ def _read_raw(raw_fpath, electrode=None, hsp=None, hpi=None, config=None,
     return raw
 
 
+def _handle_participants_reading(participants_fname, raw,
+                                 subject, verbose=None):
+    participants_tsv = _from_tsv(participants_fname)
+    subjects = participants_tsv['participant_id']
+    row_ind = subjects.index(subject)
+
+    # mapping subject information back to mne-python
+    sex_options = {'n/a': 0, 'M': 1, 'F': 2}
+    hand_options = {'n/a': 0, 'R': 1, 'L': 2, 'A': 3}
```

Comment:
these two should probably constants to be imported, because we may use them elsewhere (I think we DO already use them elsewhere 🤔 ) ... could be put into [config.py](https://github.com/mne-tools/mne-bids/blob/master/mne_bids/config.py)

Reply:
Yep we use them in write, so I moved them.